test/srt/lora/test_lora_embedding_layer.py

```diff
@@ -0,0 +1,410 @@
+# Copyright 2023-2024 SGLang Team
```

Comment:
Please add this file to here https://github.com/sgl-project/sglang/blob/849957bc76c315e18e9872a189dd9905b866769a/test/srt/run_suite.py#L24